keyboards/latinpadble/config.h

```diff
@@ -0,0 +1,103 @@
+/*
+Copyright 2021 haierwangwei2005
+
+This program is free software: you can redistribute it and/or modify
+it under the terms of the GNU General Public License as published by
+the Free Software Foundation, either version 2 of the License, or
+(at your option) any later version.
+
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License
+along with this program.  If not, see <http://www.gnu.org/licenses/>.*/
+
+#pragma once
+
+#include "config_common.h"
+#define VENDOR_ID       0x7C88 // "hw" = haierwangwei2005
+#define PRODUCT_ID      0x7C98 // "lp" = latinpadble
+#define DEVICE_VER      0x0001
+#define MANUFACTURER    haierwangwei2005
+#define PRODUCT         latinpadble
+#define DESCRIPTION     Panasonic rotary encoder BLE pad
+
+#define NO_ACTION_LAYER
+#define NO_ACTION_TAPPING
+#define NO_ACTION_ONESHOT
+#define NO_ACTION_MACRO
+#define NO_ACTION_FUNCTION
+
```

Comment:
Do you need all of these disabled?  If not, you're losing out on a lot of functionality. 

```suggestion
// #define NO_ACTION_LAYER
// #define NO_ACTION_TAPPING
// #define NO_ACTION_ONESHOT
#define NO_ACTION_MACRO
#define NO_ACTION_FUNCTION

```